lib/improver/cli/update_grid_metadata.py

```diff
@@ -61,19 +61,37 @@ def main(argv=None):
                                       'v1.1.0 meta-data')}
 
     args = ArgParser(**cli_definition).parse_args(args=argv)
+    # Load Cube
+    cube = load_cube(args.input_filepath, no_lazy_load=True)
+    # Process Cube
+    cube_changed = process(cube)
 
-    cube = load_cube(args.input_filepath)
-    cube_changed = update_stage_v110_metadata(cube)
-
+    # Save Cube
     # Create normalised file paths to make them comparable
     in_file_norm = os.path.normpath(args.input_filepath)
     out_file_norm = os.path.normpath(args.output_filepath)
     if cube_changed or in_file_norm != out_file_norm:
-        # Ensure data are not lazy in case we are writing back to the same
-        # file.
-        cube.data
         save_netcdf(cube, args.output_filepath)
 
 
+def process(cube):
+    """Update grid_id meta-data for StaGE.
+
+    Translates meta-data relating to the grid_id attribute from StaGE
+    version 1.1.0 to StaGE version 1.2.0
```

Comment:
May we well include the rest to make clear where it will and won't work:
```
Files that have no "grid_id" attribute are not recognised as v1.1.0 and are not changed. Has no effect if input_file and output_file are the same and contain a cube with non v1.1.0 meta-data.
```